src/design-system/stylesheets/components/_contact-panel.scss

```diff
@@ -25,6 +25,6 @@
 }
 
 .app-contact-panel__body {
-  @include govuk-font-regular-19
+  @include govuk-font(19, $weight: regular)
```

Comment:
Regular is the default value for `$weight` so can be omitted wherever you're calling `govuk-font`.